external/storm-cassandra/src/test/java/org/apache/storm/cassandra/DynamicStatementBuilderTest.java

```diff
@@ -56,78 +61,93 @@
         when(mockTuple.getFields()).thenReturn(new Fields("weatherstation_id", "event_time", "temperature"));
```

Comment:
"weatherstation_id", "event_time", "temperature" can be declared as constants and used across the class.